app/helpers/application_helper.rb

```diff
@@ -164,4 +164,45 @@ def yes_no_span(boolean_value)
     boolean_value ? content_tag(:span, t('yes'), class: 'yes') : content_tag(:span, t('no'), class: 'no')
   end
 
+
+  # Create and return a span tag with a FontAwesome icon and data-toggle: 'tooltip'
+  # Default FontAwesome icon is 'fas.fa-info-circle'
+  # Example: fas_tooltip("This is the text when the user hovers over the icon")
+  #   will return
+  #    '<span class="i fas fa-info-circle"
+  #        title=""
+  #        data-original-title="This is the text when the user hovers over the icon">
+  #        data-toggle="tooltip"
+  #     </span>'
+  #
+  # Example: fas_tooltip("tooltip text", fa_icon: 'calendar-alt')
+  #   will return
+  #    '<span class="i fas fa-calendar-alt"
+  #        title=""
+  #        data-original-title="tooltip text">
+  #        data-toggle="tooltip"
+  #     </span>'
+  #
+  # Example: fas_tooltip(I18n.t('share_on_facebook'), fa_icon_group: 'fab', fa_icon: 'facebook')
+  #   will return
+  #    '<span class="i fab fa-facebook"
+  #        title=""
+  #        data-original-title="Share on Facebook">
+  #        data-toggle="tooltip"
+  #     </span>'
+  #
+  #
+  # @param title [String] - the
```

Comment:
here may be words missing?

Reply:
yup - good catch. :-)  I'll finish that up in a bit.